content/contributor/social-service/2-login.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Login
+url: /contributor/social-service/login
+weight: 2
+---
+
+# Login to Social Service
+
+## Description
+
+The authentication process for the Social Service involves interacting with the Matrix server. Users must log in to Matrix using their address, signing a fetch operation to obtain an authentication token. This token is then used to authorize interactions with the Social Service.
+
+## HTTP Request
+
+To acquire the authentication token, a POST request must be made to the following URL: `https://social.decentraland.org/_matrix/client/r0/login`
+
+### Request Body
+
+```json
+{
+  "auth_chain": authChain,
+  "identifier": {
+    "type": "m.id.user",
+    "user": address
+  },
+  "timestamp": timestamp.toString(),
+  "type": "m.login.decentraland"
+}
+```
+
+### Response Body
+
+```json
+{
+  "user_id": "@0x123abC:decentraland.org",
+  "social_user_id": "0x123abC",
+  "access_token": "syt_SomETokEN",
+  "device_id": "FRFREGRG",
+  "home_server": "decentraland.org",
+  "well_known": {
+      "m.homeserver": {
+          "base_url": "https://synapse.decentraland.org/"
+      }
+  }
+}
+```
+
+The authentication token, present in the access_token field, is required for subsequent interactions with the Social Service.
+
+## JS Code Example
+
+```javascript
+import fetch from 'cross-fetch'
+import { AuthIdentity, Authenticator } from '@dcl/crypto'
+
+export async function getSynapseToken(synapseUrl: string, address: string, identity: AuthIdentity): Promise<string> {
```

Comment:
We could mention somewhere what Synapse is or rename these to Matrix to avoid confusion.